core/src/apps/management/change_pin.py

```diff
@@ -50,7 +50,7 @@ def require_confirm_change_pin(ctx: wire.Context, msg: ChangePin) -> None:
     if not msg.remove and has_pin:  # changing pin
         text = Text("Change PIN", ui.ICON_CONFIG)
         text.normal("Do you really want to")
-        text.bold("change the current PIN?")
+        text.bold("change your PIN?")
```

Comment:
Reason: 1. Overflow. 2. Elsewhere we say _your_ PIN rather than _current_ PIN, e.g. "Enter your PIN".

<img src="https://user-images.githubusercontent.com/42678794/65316086-8dfa9680-db99-11e9-836a-11a242669c64.png" width=300><img src="https://user-images.githubusercontent.com/42678794/65316194-c26e5280-db99-11e9-982c-777ee097168c.png" width=300>